Dockerfile

```diff
@@ -5,6 +5,8 @@ LABEL Description="Cutting-edge LAMP stack, based on Ubuntu 16.04 LTS. Includes
 	Usage="docker run -d -p [HOST WWW PORT NUMBER]:80 -p [HOST DB PORT NUMBER]:3306 -v [HOST WWW DOCUMENT ROOT]:/var/www/html -v [HOST DB DOCUMENT ROOT]:/var/lib/mysql fauria/lamp" \
 	Version="1.0"
 
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
```

Comment:
Will this script be available in the long run?

Also, we are installing nodejs and npm on line 49 and then doing some linking below. We may need to change that to avoid having multiple instances of node.